proto/topo.proto

```diff
@@ -81,6 +81,34 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host vm constraints required by the vendor.
+  repeated HostConstraint host_constraints = 13;
+}
+
+// HostConstraint includes all the necessary system constraints required 
+// which could be a sysctl setting/cgroup
+message HostConstraint{
+  oneof constraints{
+    // Includes the required sysctl settings that need to be configured
+    Sysctl sysctl_constraint = 1;
+  }
+}
+
+// Sysctl settings for the host
+message Sysctl{
```

Comment:
I wonder if these are best described as KernelParameter or similar as sysctl is just the cli tool used to configure the kernel params

Reply:
Fixed.